quic/s2n-quic/src/provider/endpoint_limits.rs

```diff
@@ -29,6 +34,125 @@ impl<T: 'static + Limiter> Provider for T {
     }
 }
 
+const THROTTLED_PORT_LIMIT: usize = 10;
+const THROTTLE_FREQUENCY_MS: u64 = 1000;
```

Comment:
It's better to store this as a `Duration` so you don't have to specify/remember the units:

```suggestion
const THROTTLE_FREQUENCY: Duration = Duration::from_secs(1);
```

Reply:
Didn't realize `Duration` could be a const! Thanks!